### PR TITLE
chore(StatusQ): Backpressure files deduplicated, imports adjusted

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

- Removes `Backpressure` utility from `ui/imports/utils/Backpressure` as it's already in `StatusQ`
- Adjusts imports
- Removes methods not used and not working because not met dependency on `QuickPromise`
- Do minor refactor, mainly `var` -> `let`/`const`

Closes: #15558

### Affected areas
Files using `Backpressure` utility.

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml [hunk 1]
```diff
@@ -10,13 +10,15 @@ import shared.panels 1.0
 import shared.stores 1.0 as SharedStores
 
 import StatusQ 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Controls.Validators 0.1
 import StatusQ.Core 0.1
+import StatusQ.Core.Backpressure 0.1
```
Comment (alexjba) [inline ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml:17]:
```suggestion
```

**Review by alexjba — APPROVED** [review]
Duplicated import in one file!
LGTM otherwise! 👍

File: ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml (hunk 1 above)
Comment (micieslak) [reply ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml:17]: OMG! Thanks for catching that!

**Pull request merged**